third_party/cyw30739_sdk/gen_ota_image.py

```diff
@@ -84,5 +99,89 @@ def pad_data(data, aligned_size):
     return data
 
 
+def gen_image(option, ds_header, ds_data, xs_header, cx_data):
+
+    configs = parse_config(option)
+
+    write_binary(option, ds_header, ds_data, xs_header, cx_data)
+    run_ota_image_tool(option, configs)
+
+    # Get the header size
+    header_size = 0
+    for line in subprocess.run(
+        [option.ota_image_tool, "show", option.binary.with_suffix(".ota"), ],
+        check=True,
+        capture_output=True,
+        text=True,
+    ).stdout.splitlines():
+        if line.startswith("Header Size:"):
+            header_size = int(line.split(":")[1])
+            break
+
+    if header_size % 4 == 0:
+        return 0
```

Comment:
is there any value in returning anything here if this method always returns 0?

Reply:
fixed.